### PR TITLE
[components-webview] Add image share option to context menu. Fixes JB#56905

### PR DESCRIPTION
**Pull request by mkenttala**

### CODE REVIEW

File: import/popups/ContextMenu.qml [hunk 1]
```diff
@@ -211,6 +211,17 @@ ContextMenuInterface {
...
             }
 
+            MenuItem {
+                visible: root.isImage && root.url
+                //: Copy image address to clipboard from context menu
+                //% "Copy image address"
+                text: qsTrId("sailfish_components_webview_popups-me-copy_address_to_clipboard")
+                onClicked: {
+                    root._hide()
+                    Clipboard.text = root.url
```
Comment (rainemak) [inline import/popups/ContextMenu.qml:221]: This should be ShareAction to share an image. Similar to share option in the browser menu to share a url.

File: import/popups/ContextMenu.qml [hunk 2]
```diff
@@ -211,6 +211,17 @@ ContextMenuInterface {
                 onClicked: root._hide()
             }
 
+            MenuItem {
+                visible: root.isImage && root.url
+                //: Copy image address to clipboard from context menu
+                //% "Copy image address"
```
Comment (rainemak) [inline import/popups/ContextMenu.qml:217]: Just "Share" as the context indicates already that you long pressed on top of an image.

Comment (rainemak) [reply import/popups/ContextMenu.qml:217]: You could enhance WebShareAction to take option share menu title. Currently it's always "Share link".

See: 
https://github.com/sailfishos/sailfish-components-webview/pull/155/files#diff-9452e87a299d714c5663501930922ad070ab060fc3b90f654317fac5bacc7ba4R146

**Review by rainemak — APPROVED** [review]

**Review by llewelld — APPROVED** [review]
LGTM also.

**Pull request merged**